src/parser/parser.yy

```diff
@@ -2039,21 +2044,12 @@ admin_job_sentence
     }
     ;
 
-admin_job_operation
-    : KW_COMPACT         { $$ = new std::string("compact"); }
-    | KW_FLUSH           { $$ = new std::string("flush"); }
-    | KW_REBUILD KW_TAG  { $$ = new std::string("rebuild tag"); }
```

Comment:
Don't support index job?

Reply:
Support it. Rebuid index is in `rebuild_tag_index_sentence`, maybe move it into `admin_job_operation` is better?

Reply:
IMO. The rebuild sentence will not maintain again  and it's reserved only for compatibility. Rebuild by job is the better way.

Reply:
Good idea. We need to have a discussion on it.